testscript.sh

```diff
@@ -0,0 +1 @@
+git rev-parse --show-toplevel
```

Comment:
why this new file? Looks like accidental commit

Reply:
Oh no, my bad-was testing if it would be feasible to use the suggestion you had provided, while I was actively using this branch for development